### DISCUSSION TITLE
Sayı çifti sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#288] [Zeka Sorusu]
14 91 62 53 64 96 48 11 00
Sonraki sayı çifti nedir?

**Comment by AhmetEmirErsoz** [comment]
1,4,9,16,25,36,49,64,81,100,121 çünkü 1 den 11 e kadar olan sayıların üslü halleri 1 in karesi 1 2 nin 4, 3ün 9 diye devam ediyor
